Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/TokenControllerUsingJsonTests.cs

```diff
@@ -163,1448 +335,1010 @@ public void Should_use_ClientAppRepo_to_create_token_using_the_supplied_ApiClien
-
-        //        _actualJsonContent = JObject.Parse(actualContent);
-        //    }
-
-        //    [Assert]
-        //    public void Should_return_HTTP_status_of_BadRequest()
-        //    {
-        //        Assert.That(_actualResponseMessage.StatusCode, Is.EqualTo(HttpStatusCode.BadRequest));
-        //    }
-
-        //    [Assert]
-        //    public void Should_return_a_single_valued_response_with_an_error_indicating_invalid_client()
-        //    {
-        //        Assert.That(
-        //            _actualJsonContent.Properties()
-        //                .Count(),
-        //            Is.EqualTo(1),
-        //            _actualJsonContent.ToString());
-
-        //        Assert.That(
-        //            _actualJsonContent["error"]
-        //                .Value<string>(),
-        //            Is.EqualTo("invalid_client"));
-        //    }
-
-        //    [Assert]
-        //    public void Should_not_include_any_cache_headers()
-        //    {
-        //        Assert.That(_actualResponseMessage.Headers.CacheControl, Is.Null);
-        //        Assert.That(_actualResponseMessage.Headers.Pragma, Is.Empty);
-        //    }
-
-        //    public override void RunOnceAfterAll()
-        //    {
-        //        _clientAppRepo.VerifyAllExpectations();
-        //    }
-        //}
+        public class With_a_scope_matching_an_associated_EdOrgId : TestFixtureAsyncBase
+        {
+            private IClientAppRepo _clientAppRepo;
+            private IApiClientAuthenticator _apiClientAuthenticator;
+            private TokenController _controller;
+
+            private ApiClient _suppliedClient;
+            private Guid _suppliedAccessToken;
+            private string _requestedScope;
+
+            private IActionResult _actionResult;
+            private TokenResponse _tokenResponse;
+
+            protected override Task ArrangeAsync()
+            {
+                _suppliedClient = new ApiClient
+                {
+                    ApiClientId = 1,
+                    ApplicationEducationOrganizations = new List<ApplicationEducationOrganization>()
+                    {
+                        new ApplicationEducationOrganization {EducationOrganizationId = 997},
+                        new ApplicationEducationOrganization {EducationOrganizationId = 998},
+                        new ApplicationEducationOrganization {EducationOrganizationId = 999},
+                    }
+                };
+
+                _apiClientAuthenticator = Stub<IApiClientAuthenticator>();
+
+                // Scope the request to the first associated EdOrg
+                _requestedScope = _suppliedClient.ApplicationEducationOrganizations
+                    .Select(x => x.EducationOrganizationId)
+                    .First()
+                    .ToString();
+
+                _suppliedAccessToken = Guid.NewGuid();
+
+                _clientAppRepo = Stub<IClientAppRepo>();
+
```

Comment:
formatting issue, remove xtra blank lines (should be 1)

Reply:
Fixed